testomania/src/main/java/com/earth/testomania/home_screen/presentation/ui_components/GridWithItems.kt

```diff
@@ -71,16 +75,35 @@ fun CardButton(
     scaffoldState: ScaffoldState,
     modalBottomSheetState: ModalBottomSheetState,
     networkConnectivityObserver: NetworkConnectivityObserver,
+    addBottomPadding: Boolean
 ) {
     val scope = rememberCoroutineScope()
     val status by networkConnectivityObserver.observe().collectAsState(initial = false)
 
+    val infoText = stringResource(R.string.check_your_connection)
+    val dismissText = stringResource(R.string.dismiss)
+
+    var tileHeight = 125.dp
+    var tileBottomPadding = 0.dp
```

Comment:
I think it's better to be immutable

Reply:
Cant, cause I am change the values below:
```
var tileHeight = 125.dp
  var tileBottomPadding = 0.dp
  if (addBottomPadding) {
      tileHeight = 170.dp
      tileBottomPadding = 45.dp
  }
```